doc/rst/source/explain_-aspatial.rst_

```diff
@@ -1,2 +1,2 @@
-**-a**\ [*col*\ =]\ *name*\ [,\ *...*] :ref:`(more ...) <-aspatial_full>`
+**-a**\ [[*col*\ =]\ *name*\ [,\ *...*]] :ref:`(more ...) <-aspatial_full>`
```

Comment:
Can **-a** really be used without *col*=*name*?

Reply:
That is what the docs say ...

Reply:
I interpret the quote below from the docs as at least one instance of *col*=*name* is required, but this change in the PR indicates that it is optional.

> give one or more comma-separated associations col=name